### PR TITLE
Remove pip installation

### PR DESCRIPTION
**Pull request by DLu**
Can you review @reinzor, please?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,12 +41,13 @@ You also need to add a dependency on the `tf_transformations` in your `package.x
...
+This package is available in full as a ROS binary, which you can install via
 
-    sudo pip3 install transforms3d
+    sudo apt install ros-$ROSDISTRO-tf-transformations
+
+In older versions, you would need to manually install the `transforms3d` library via pip.
 
-At the present time, you cannot install `transforms3d` via Ubuntu debian, nor can you force the installation of the
- `tf_transformations` package to also include `transforms3d`.
+    sudo pip3 install transforms3d
```
Comment (destogl) [inline README.md:50]: @DLu I had an issue yesterday, and it appears that the library expects explicit version of `transforms3d`, I have ended up having version `0.4.2` on Ubuntu 22.04 (within Docker).

This caused the issue that `tf_transforms` didn't recognize the installation. Then I have installed explicit the version `0.4.1` and everything worked.

```suggestion
    sudo pip3 install transforms3d==0.4.1
```

**Comment by reinzor** [comment]
@Timple can you follow up on this; I think you have more insight in the issue.

File: README.md [hunk 2]
```diff
@@ -16,15 +16,15 @@ However, ROS 2 only supports [TF2](https://wiki.ros.org/tf2) (not to be confused
...
 
 However, using that library has a few obstacles that make porting ROS 1 code to ROS 2 difficult.
  1. The API is different. The new API has more consistent naming, but even then, it is not a one-to-one translation. for example, `tf.transformations.quaternion_from_euler` could be replaced with `transforms3d.euler.euler2quat`, but `tf` returns the quaternion with the ordering `x, y, z, w` and `transforms3d` returns `w, x, y, z`.
  2. Not all of the functions have been migrated. As noted in the [`transforms3d` Refactoring Plan](https://matthew-brett.github.io/transforms3d/devel/refactor_plan.html), some of the functions are still in the "to integrate" pile.
 
 ## Migration
 If you're here, its likely because you want to migrate some code easily. You have two options:
- 1. Use `transforms3d` by adding a dependency on `python-transforms3d-pip` in your `package.xml` and noting differences in API discussed above.
+ 1. Use `transforms3d` by adding a dependency on `python3-transforms3d` in your `package.xml` and noting differences in API discussed above.
```
Comment (Timple) [inline README.md:27]: This dependency is not required on the external package. It should come in depending only on this package.

File: README.md (hunk 1 above)
Comment (Timple) [reply README.md:50]: This should not be the case.
Can you provide a minimal dockerfile which has this issue?

I believe the issue might come from the fact that you actually did install it via pip at one point in time.

**Review by Timple — CHANGES_REQUESTED** [review]

**Comment by Timple** [comment]
I've replicated the issue in a dockerfile:

```Dockerfile
FROM ros:iron
SHELL [ "/bin/bash", "-c" ]
RUN apt-get update -qq && apt-get install -qqy ros-iron-tf-transformations
RUN source /opt/ros/iron/setup.bash && python3 -c "from tf_transformations import euler_from_quaternion"
```

The issue is that the buildfarm still uses the old package.xml.

@DLu can you do a new bloom release? Because https://github.com/DLu/tf_transformations/pull/10 is still not part of the current release.

**Pull request merged**